openpype/plugins/inventory/remove_and_load.py

```diff
@@ -0,0 +1,43 @@
+from openpype.pipeline import InventoryAction
+from openpype.pipeline.legacy_io import Session
```

Comment:
```suggestion
from openpype.pipeline import get_current_project_name
```

Reply:
May I ask for the reason of this suggestion? Perhaps `legacy_io` is deprecated? I'm fine with these changes, I am just curious because if what I did here should be avoided, then I need to change my practices.

Reply:
`legacy_io` is not yet deprecated, we want to make it deprecated, we can't until all usages are removed and replaced. Most of changes in core from last 2-3 releases reduced it's usage to minimum.

Reply:
Okay, it makes sense. I'll try to reduce my use of `legacy_io` from now on.